test/util/generator_helpers.rb

```diff
@@ -0,0 +1,150 @@
+require "yaml"
+
+require 'beaker-hostgenerator'
+require 'beaker-hostgenerator/data/vmpooler'
+require 'beaker-hostgenerator/data'
+
+module GeneratorTestHelpers
+  include BeakerHostGenerator::Data
+  include BeakerHostGenerator::Data::Vmpooler
+
+  def run_cli_with_options options = []
+    STDERR.reopen("stderr.txt", "w")
+    cli = BeakerHostGenerator::CLI.new(options)
+    yaml_string = cli.execute
+
+    hash = YAML.load(yaml_string)
+    return hash
```

Comment:
Whoa, forgot this was a keyword! Looks like most code doesn't bother using `return`, I'd probably remove it but no big deal